specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-12-01-preview/types/faultSimulations.json

```diff
@@ -0,0 +1,150 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "1.0",
+      "title": "Common types used by Resource Providers to implement standard fault simulation actions on resource-types"
+    },
+    "paths": {},
+    "definitions": {
+      "FaultKind": {
```

Comment:
should we change it to FaultType? I used kind following some convention in our code but I think type is more common